ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -50,6 +52,8 @@
                     FLAVOR=notcmalloc
       - conditional-step:
           condition-kind: shell
+          # Only run for mimic and nautilus, explicitly skip (do not run) on jewel, kraken,
+          # and luminous
           condition-command: |
             echo "${GIT_BRANCH}" | grep -v '\(jewel\|kraken\|luminous\)' | grep '\(mimic\|nautilus\)'
```

Comment:
I know this isn't part of this PR but why is the `grep -v '\(jewel\|kraken\|luminous\)'` even necessary here?

Reply:
maybe not, but I don't want to touch these really

Reply:
@djgalloway no, it's not necessary. will fix it in a follow-up PR.